### DISCUSSION TITLE
Problem sending push notifications

### DISCUSSION THREAD

**Discussion by GiordanoVicoli** [relayrides/pushy#984] [Q&A]
Hi,

I tried pushy to send push notifications but I have this message : 
Notification rejected by the APNs gateway: Optional[TopicDisallowed]

Is there someone can help me?

Thanks and regards,
Giordano

**Comment by jchambers** [comment]
Well, it sounds like you're trying to send to an APNs topic that isn't covered by your current set of credentials. Your topic should, under most circumstances, match your app's bundle ID.

If you need help, could you please tell us more about:

- What are you using for credentials? A certificate? A signing key?
- Are you sure the topic is correct?
- What have you tried to solve this problem on your own?

Thanks.

**Reply by GiordanoVicoli** [reply]
Hi,

thank you very much for the answer and the suggestions. 
However I solved the problem, I missed a detail in the configuration phase.

Thanks again and regards,
Giordano

**Comment by ylx19931228** [comment]
hi, 
I am currently using the tool and found that I also have this problem.
Project configuration figure.
![微信截图_20241209191838](https://github.com/user-attachments/assets/69a7d4a2-1c15-417c-9521-6b34756a2116)

Notification rejected by the APNs gateway: Optional[TopicDisallowed]

An error was found when using swagger calls. How do I deal with it?

**Reply by ylx19931228** [reply]
![微信截图_20241209191859](https://github.com/user-attachments/assets/e34cbe7d-de28-4b8f-8a74-453ea2463d4a)
